stories/topics/tech-note-gcp-controller-api.adoc

```diff
@@ -0,0 +1,5 @@
+[id="tech-note-gcp-controller-api"]
+
+= Ansible Automation Platform Controller API
+
+API endpoints for Controller need to be suffixed with `/` in order for requests to go through. For example a request such as `<controller_base_url>/api/v2/metrics` will timeout while `<controller_base_url>/api/v2/metrics/` will go through.
```

Comment:
must be suffixed with / for requests to go through   (/ in backticks)

For example, a request such as .... times out while .... goes through.